calculator.rb

```diff
@@ -0,0 +1,111 @@
+# If the result is a whole number, program should return an integer. Otherwise, it should return a float rounded to 2 decimal points.
+def checkForDecimal(result)
+  if result % 1 == 0
+    result = result.to_i
+  else
+    result = result.to_f.round(2)
+  end
+end
+
+# Methods for each operator
+def add(num1, num2)
+  result = num1 + num2
+  checkForDecimal(result)
+end
+
+def subtract(num1, num2)
+  result = num1 - num2
+  checkForDecimal(result)
+end
+
+def multiply(num1, num2)
+  result = num1 * num2
+  checkForDecimal(result)
+end
+
+def divide(num1, num2)
+  result = num1 / num2
+  checkForDecimal(result)
+end
+
+def exponent(num1, num2)
+  result = num1 ** num2
+  checkForDecimal(result)
+end
+
+def modulo(num1, num2)
+  result = num1 % num2
+  checkForDecimal(result)
+end
+
+# Methods for getting user numbers. Proram should keep prompting user until they enter an integer.
+def getFirstUserNum
+  begin
+    return Float(gets.chomp)
+  rescue
```

Comment:
Good work figuring out the `begin`/`rescue` syntax. We'll talk more about what this is in a couple weeks.